docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -57,6 +57,29 @@ You can use the YugabyteDB Anywhere UI to enable LDAP authentication for Yugabyt
 
   - The **Bind DN** field value represents the distinguished name (DN) used for searching and binding.
 
+### Map LDAP groups to YugabyteDB Anywhere roles
```

Comment:
Would you please create a new section called "Role Management with LDAP" and move this, as well as the section on yugabytePlatformRole into that?